benchmarks/_sources.py

```diff
@@ -459,6 +468,9 @@ def table(self):
                 parse_options=self.csv_parse_options,
                 convert_options=self.csv_convert_options,
             )
+
+            self._feather_write(self._table, path, compression="lz4")
+
```

Comment:
Is this needed here? It seems out of place?

Reply:
It's not wholly, but it will prevent us from rereading the CSVs repeatedly. A few lines up uses this file instead if it exists, which for big files will be a lot faster, and without this, it will only get generated if a benchmark that uses it gets run.